libkbfs/mdserver_tlf_storage.go

```diff
@@ -5,52 +5,77 @@
 package libkbfs
 
 import (
-	"bytes"
-	"encoding/binary"
 	"errors"
 	"fmt"
 	"io/ioutil"
 	"os"
 	"path/filepath"
 	"sync"
 
-	"golang.org/x/net/context"
-
-	"github.com/syndtr/goleveldb/leveldb"
-	"github.com/syndtr/goleveldb/leveldb/util"
+	keybase1 "github.com/keybase/client/go/protocol"
 )
 
+// mdServerTlfStorage stores an ordered list of metadata IDs for each
+// branch of a single TLF, along with the associated metadata objects,
+// in flat files on disk.
+//
+// The directory layout looks like:
+//
+// dir/md_branch_journals/00..00/EARLIEST
+// dir/md_branch_journals/00..00/LATEST
+// dir/md_branch_journals/00..00/0...001
+// dir/md_branch_journals/00..00/0...002
+// dir/md_branch_journals/00..00/0...fff
+// dir/md_branch_journals/00..cc/EARLIEST
```

Comment:
This makes it look like branch IDs are deterministic?  Or something?  Maybe the non-zero one should have random bytes instead of `00..cc`.